.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blank_issues_enabled: false
+contact_links:
+  - name: Question? 
+    url: https://github.com/JanusGraph/janusgraph/discussions
+    about: Who, what, when, where, and how? Use GitHub discussions.
+  - name: Issue with JanusGraph Docker image
+    url: https://github.com/JanusGraph/janusgraph-docker/issues/new
+    about: Please open issues relating to Docker Image in janusgraph-docker.
+  - name: Issue with JanusGraph FoundationDB Adapter
+    url: https://github.com/JanusGraph/janusgraph-foundationdb
```

Comment:
(nitpick) shall we use "https://github.com/JanusGraph/janusgraph-foundationdb/issues/new"? Since you are using this for the janusgraph-docker link above.